### PR TITLE
(DOCS) Add release notes for 6.11.0

### PR DESCRIPTION
**Pull request by Zak-Kent**

### CODE REVIEW

**Comment by Zak-Kent** [comment]
@rbrw could you take a look at the TODO in the new features section on Monday? I'm not sure what if anything we want to mention related to command broadcast. 

@clairecadman this should be mostly ready for a second set of eyes when you get the chance.

File: documentation/release_notes/release_notes_latest.markdown [hunk 1]
```diff
@@ -21,6 +21,30 @@ canonical: "/puppetdb/latest/release_notes.html"
...
+significantly speed up the migration included in this release.
+
+### New features
+ - **Added support for storage of reports generated by Bolt Plans** Bolt Plans will start submitting reports at a later date
+
+ - **New report field `type`. Specifies the type of report submitted.** Either 'agent' for a puppet agent run, or 'plan' for a report of a plan's apply block
+
+ - #TODO figure out what we want to mention about command broadcast
+
+### Notes
```
Comment (austb) [inline documentation/release_notes/release_notes_latest.markdown:40]:
```suggestion
### Deprecations
```
I feel like people might skip reading the notes

Comment (Zak-Kent) [reply documentation/release_notes/release_notes_latest.markdown]: 👍 updated

File: documentation/release_notes/release_notes_latest.markdown [hunk 2]
```diff
@@ -21,6 +21,30 @@ canonical: "/puppetdb/latest/release_notes.html"
...
+### New features
+ - **Added support for storage of reports generated by Bolt Plans** Bolt Plans will start submitting reports at a later date
+
+ - **New report field `type`. Specifies the type of report submitted.** Either 'agent' for a puppet agent run, or 'plan' for a report of a plan's apply block
+
+ - #TODO figure out what we want to mention about command broadcast
+
+### Deprecations
+   - Java 8, 9, and 10 have been deprecated and PuppetDB will now log warnings on startup if these version of Java are being used. We recommend using Java 11 going forward
+   - Running PuppetDB with PostgreSQL 9.6 and 10 has been deprecated, please use PostgreSQL 11
```
Comment (clairecadman) [inline documentation/release_notes/release_notes_latest.markdown:42]:
```suggestion
   - Running PuppetDB with PostgreSQL 9.6 and 10 has been deprecated. Use PostgreSQL 11 instead. 
```

File: documentation/release_notes/release_notes_latest.markdown [hunk 3]
```diff
@@ -21,6 +21,30 @@ canonical: "/puppetdb/latest/release_notes.html"
...
+
+### New features
+ - **Added support for storage of reports generated by Bolt Plans** Bolt Plans will start submitting reports at a later date
+
+ - **New report field `type`. Specifies the type of report submitted.** Either 'agent' for a puppet agent run, or 'plan' for a report of a plan's apply block
+
+ - #TODO figure out what we want to mention about command broadcast
+
+### Deprecations
+   - Java 8, 9, and 10 have been deprecated and PuppetDB will now log warnings on startup if these version of Java are being used. We recommend using Java 11 going forward
```
Comment (clairecadman) [inline documentation/release_notes/release_notes_latest.markdown:41]:
```suggestion
   - Java 8, 9, and 10 have been deprecated. If these versions are used, PuppetDB will log a warning on startup. We recommend using Java 11 going forward.
```

File: documentation/release_notes/release_notes_latest.markdown [hunk 4]
```diff
@@ -21,6 +21,30 @@ canonical: "/puppetdb/latest/release_notes.html"
...
+
+### Upgrading
+
+We recommend upgrading to PostgreSQL 11 or greater before upgrading to
+PuppetDB 6.11.0. PostgreSQL 11 includes performance improvements which make
+adding a non-null column with a default value much faster and should
+significantly speed up the migration included in this release.
+
+### New features
+ - **Added support for storage of reports generated by Bolt Plans** Bolt Plans will start submitting reports at a later date
```
Comment (clairecadman) [inline documentation/release_notes/release_notes_latest.markdown:34]:
```suggestion
 - **Support for the storage of reports generated by Bolt Plans**.
```
I don't think we usually mention future features.

File: documentation/release_notes/release_notes_latest.markdown [hunk 5]
```diff
@@ -21,6 +21,30 @@ canonical: "/puppetdb/latest/release_notes.html"
...
+
+ - **New report field `type`. Specifies the type of report submitted.** Either 'agent' for a puppet agent run, or 'plan' for a report of a plan's apply block
```
Comment (clairecadman) [inline documentation/release_notes/release_notes_latest.markdown:36]:
```suggestion
 - **New report field `type` to specify the type of report submitted.** Use `agent` for a Puppet agent run, or `plan` for a report of a plans apply block
```

**Comment by Zak-Kent** [comment]
Thanks @clairecadman! Updated the pr

**Pull request merged**